src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaGiuridica/DeleghePGPagoPATest.java

```diff
@@ -130,6 +130,14 @@ public void nellaSezioneDelegatiDallImpresaSiVisualizzaLaDelegaInStatoDiAttesaDi
         delegatiImpresaSection.controlloEsistenzaDelega(this.datiDelega.get("ragioneSociale").toString());
     }
 
+    @And("Nella sezione Delegati dall impresa si visualizza correttamente una delega in stato di attesa di conferma {string}")
+    public void nellaSezioneDelegatiDallImpresaSiVisualizzaCorrettamenteUnaDelegaInStatoDiAttesaConferma(String ragioneSociale) {
+        logger.info("Si controlla che la delega sia in stato attesa di conferma");
+
+        delegatiImpresaSection.waitLoadDelegatiImpresaPage();
```

Comment:
It doesn't check if the `table` element exist

Reply:
done